challenges/classes.js

```diff
@@ -1,7 +1,63 @@
 // 1. Copy and paste your prototype in here and refactor into class syntax.
 
+/*function CuboidMaker(attr) {
+    this.length = attr.length;
+    this.width = attr.width;
+    this.height = attr.height;
+}
+CuboidMaker.prototype.volume = function () {
+    return this.length * this.width * this.height;
+};
+CuboidMaker.prototype.surfaceArea = function () {
+    return 2 * (this.length * this.width + this.length * this.height + this.width * this.height);
+};
+*/
+
+class CuboidMakerAgain {
+	constructor(attr) {
+		this.length = attr.length;
+		this.width = attr.width;
+		this.height = attr.height;
+	}
+	volume() {
+		return this.length * this.width * this.height;
+	}
+	surfaceArea() {
+		return 2 * (this.length * this.width + this.length * this.height + this.width * this.height);
+	}
+}
+
+const cuboidAgain = new CuboidMakerAgain({
+	length: 4,
+	width: 5,
+	height: 5
+});
+
 // Test your volume and surfaceArea methods by uncommenting the logs below:
-// console.log(cuboid.volume()); // 100
-// console.log(cuboid.surfaceArea()); // 130
+console.log(cuboidAgain.volume()); // 100
+console.log(cuboidAgain.surfaceArea()); // 130
+
+// Stretch Task: Extend the base class CuboidMaker with a sub class called CubeMaker.
+//Find out the formulas for volume and surface area for cubes and create those methods using the dimension properties from CuboidMaker.
+//Test your work by logging out your volume and surface area.
+
+class CubeMaker extends CuboidMakerAgain {
+	constructor(attr) {
+		super(attr);
+	}
+	cubeVolume() {
+		return this.length ** 3;
+	}
+	cubeSurfaceArea() {
+		return this.length ** 2 * 6;
```

Comment:
as mentioned above. please use brackets to mark the separation between 2 operator.